rainier-core/src/main/scala/com/stripe/rainier/core/RandomVariable.scala

```diff
@@ -226,4 +238,85 @@ object RandomVariable {
   def fit[L, T](l: L, seq: Seq[T])(
       implicit toLH: ToLikelihood[L, T]): RandomVariable[Unit] =
     toLH(l).fit(seq)
+
+  private[core] def waic(samples: List[Array[Double]],
```

Comment:
potentially this should be broken out into a WAIC object?

Reply:
That seems reasonable, after sampling this seems separable from the RandomVariable.